website/settings/local-travis.py

```diff
@@ -42,4 +44,5 @@
 
 VARNISH_SERVERS = ['http://localhost:8080']
 
-ENABLE_VARNISH = True
+# if ENABLE_VARNISH isn't set in python read it from the env var and set it
+locals().setdefault('ENABLE_VARNISH', os.environ.get('ENABLE_VARNISH') == 'True')
```

Comment:
The setdefault lets me not reset ENABLE_VARNISH if it is set in python already. This is because the travis needs this setting False for the first two passes and True for the third.